artifactory/services/fspatterns/utils.go

```diff
@@ -30,14 +29,14 @@ func ListFiles(rootPath string, isRecursive, includeDirs, isSymlink bool, exclud
 	return filterFiles(paths, excludePathPattern)
 }
 
-// Transform to regexp and prepare Exclude patterns to be used
-func PrepareExcludePathPattern(params serviceutils.FileGetter) string {
+// Transform to regexp and prepare Exclude patterns to be used, exclusion patterns must be absolute paths.
+func PrepareExcludePathPattern(exclusions []string, patternType utils.PatternType, isRecursive bool) (string, error) {
```

Comment:
isRecursive --> recursive

This function never returned an error, so it probably shouldn't declare it.

This PR seems to be changing nothing except for this function's prototype change. Are you sure this PR is needed? Please reply here.

Reply:
yes, We now use PrepareExcludePathPattern in the ArchiveProject function in cli-core (that has been called by the GoPublish command), and GoPublishParams does not inherit from FileGetter, so I change the prototype to pass only parameters we really use in PrepareExcludePathPattern.

I'll change the return value.